inlang/source-code/paraglide/paraglide-js/src/compilerV2/compileMessage.test.ts

```diff
@@ -0,0 +1,64 @@
+import { describe, it, expect } from "vitest"
+import { compileMessage } from "./compileMessage.js"
+import type { MessageNested } from "@inlang/sdk2"
+
+describe("compileMessage", () => {
+	it("compiles a message with variants", () => {
+		const msg: MessageNested = {
+			locale: "en",
+			id: "some_message",
+			bundleId: "some_bundle",
+			declarations: [
+				{
+					type: "input",
+					name: "fistInput",
+					value: { type: "expression", arg: { type: "variable", name: "fistInput" } },
+				},
+				{
+					type: "input",
+					name: "secondInput",
+					value: { type: "expression", arg: { type: "variable", name: "second Input" } },
+				},
+			],
+			selectors: [
+				{ type: "expression", arg: { type: "variable", name: "fistInput" } },
+				{
+					type: "expression",
+					arg: { type: "variable", name: "second Input" },
```

Comment:
This can't work? The input name has whitespace which does not match the declarations. Hmmm supports the "we should enforce snake_case for names" argument. Do you agree?

Reply:
Having spaces in variable names is perfectly ok. The space in the test is on purpose to make sure it works as expected